doc/directions.md

```diff
@@ -0,0 +1,2118 @@
+                                        [
+                                            2.343005, 
+                                            48.889877
+                                        ], 
+                                        [
+                                            2.343078, 
+                                            48.889975
+                                        ]
+                                    ], 
+                                    "type": "LineString"
+                                }, 
+                                "info": null, 
+                                "maneuver": {
+                                    "instruction": "Tourner à gauche sur la rue du Mont Cenis", 
+                                    "location": [
+                                        2.342943, 
+                                        48.889792
+                                    ], 
+                                    "modifier": "left", 
+                                    "type": "turn"
+                                }, 
+                                "mode": "BICYCLE", 
+                                "stops": []
+                            }, 
+                            {
+                                "distance": 0, 
+                                "duration": 0, 
+                                "geometry": {
+                                    "coordinates": [
+                                        [
+                                            2.343078, 
+                                            48.889975
+                                        ], 
+                                        [
+                                            2.343078, 
+                                            48.889975
+                                        ]
+                                    ], 
+                                    "type": "LineString"
+                                }, 
+                                "info": null, 
+                                "maneuver": {
+                                    "instruction": "Vous êtes arrivé à votre destination, sur la gauche", 
+                                    "location": [
+                                        2.343078, 
+                                        48.889975
+                                    ], 
+                                    "modifier": "left", 
+                                    "type": "arrive"
+                                }, 
+                                "mode": "BICYCLE", 
+                                "stops": []
+                            }
+                        ], 
+                        "summary": "Rue Saule, Rue Caulaincourt"
+                    }
+                ], 
+                "price": null, 
+                "summary": null
+            }
+        ]
+    }, 
+    "status": "success"
+}
+```
+
+### `publictransport`
+
+*Some geometries are simplified for readability*
+
+```json
```

Comment:
You should put your code examples between `<details>` html tags. This is just too big as is.

Reply:
done